Portfolio/index.html

```diff
@@ -8,6 +8,59 @@
 
   <body style="margin: 0; padding: 0">
     <!-- Write your code below for PORTFOLIO -->
+   <h1>Chauhan.</h1>
+    <ul>
+      <li>ABOUT</li>
+      <li>SKILLS</li>
+      <li>PROJECTS</li>
+      <LI>REVIEWS</LI>
+    </ul><br>
+  <img src="/phone.icon.png" alt="Dialer Logo"/><br>
+  <button>CONTACT ME</button><br>
+
+<section>
+   <h1>Abhishek Chauhan<h1><br>
+        <img src="/johndoe.png" alt="Profile Picture"/><br>
+    <h1>Front End<br> Developer</h1>
+    <h3>GET TO KNOW ME </h3>
+        <p>My name is Abhishek, and I'm a passionate front-end developer with 2 Years of experience crafting responsive and user-friendly websites and applications. 
+        I specialize in [specific technologies or frameworks, e.g., React, Vue, Angular], and I have a strong grasp of the modern web development ecosystem including HTML, CSS, JavaScript, and related tools.</p><br>
+    <ul>
+          <li>Name:Abhishek Chauhan</li>
+          <li>Age:20</li>
+          <li>Experience:2years</li>
+  </section>
+    <h2> SKILLS</h2>
+          <img src="/html.icon.png" alt="html logo"/><br>
+          <img src="/css.icon.png" alt="css logo"/><br>
+          <img src="/js.icon.png" alt="js logo" /><br>
+          <img src="/ts.icon.png" alt="ts logo"/><br>
+          <img src="/svelte.icon.png" alt="svelte"/><br>
+          <img src="/firebase.icon.png" alt="firebase logo"/><br>
+          <img src="/react.icon.png" alt="react logo"/><br>
+          <h3> GET IN TOUCH </h3>
+          <h3> CONTACT</h3>
+    <form action="mailto:abhishekchauhan4871@gmail.com" method="GET">
+      <label>Name:</label>
+      <input type="text" name="Name"><br>
+      <label>E-mail:</label>
+      <input type="text" name="email_addr" value=""><br>
+      <label for="message">Your Message:</label>
+     <textarea id="message" name="message" rows="4" cols="50" placeholder="Type your message here..."></textarea>
+
```

Comment:
The closing tag on line 43 should be lowercase '/h4' instead of uppercase '/h4'.